.github/workflows/ci-ptf-tests.yml

```diff
@@ -0,0 +1,90 @@
+    P4C_DIR: ${{ github.workspace }}/p4c
+    SDE_INSTALL: ${{ github.workspace }}/install
+    IPDK_RECIPE: ${{ github.workspace }}/ipdk.recipe
+    DEPEND_INSTALL: ${{ github.workspace }}/install
+
+jobs:
+  build_p4dpdk_ubuntu:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: 'Checkout DPDK-target'
+        uses: actions/checkout@v3
+        with:
+             repository: p4lang/p4-dpdk-target
+             path: p4sde
+             submodules: 'recursive'
+
+      - name: 'Checkout ipdk-recipe'
+        uses: actions/checkout@v3
+        with:
+             repository: ipdk-io/networking-recipe
+             path: ipdk.recipe
+             submodules: 'recursive'
+
+      - name: checkout P4C
+        uses: actions/checkout@v3
+        with:
+             path: p4c
+             submodules: recursive
+
+      - name: 'Install DPDK dependencies'
+        working-directory: p4sde/tools/setup
+        run: |
+              sudo apt update -y
+              python install_dep.py
+
+      - name: 'Compile p4sde dpdk target'
+        working-directory: p4sde
+        run: |
+             mkdir ${GITHUB_WORKSPACE}/install
+             ./autogen.sh
+             ./configure --prefix=$SDE_INSTALL
+             make 
+             make install
+
+      - name: Build p4c with only the DPDK backend
+        working-directory: p4c
+        env:
+          OUTPUT_DIR: ${{ github.workspace }}/p4c/tools/ci-dpdk-ptf-test/ptf-simple-test/add_on_miss0
+        run: |
+          sudo -E tools/ci-dpdk-ptf-test/ci-build.sh
+
+      - name: 'Compile ipdk recipe'
+        run: |
+             echo "Install infrap4d dependencies"
+             cd ipdk.recipe
+             sudo apt install libatomic1 libnl-route-3-dev openssl
+             sudo pip3 install -r requirements.txt
+             cd $IPDK_RECIPE/setup
+             echo "Build infrap4d dependencies"
+             cmake -B build -DCMAKE_INSTALL_PREFIX="$SDE_INSTALL" -DUSE_SUDO=ON
+             cmake --build build 
+             echo "Build Networking Recipe"
+             source $IPDK_RECIPE/scripts/dpdk/setup_env.sh $IPDK_RECIPE $SDE_INSTALL $DEPEND_INSTALL 
+             cd $IPDK_RECIPE
+             sudo ./make-all.sh --target=dpdk --no-krnlmon --no-ovs -S $SDE_INSTALL -D $DEPEND_INSTALL
+      
+      - name: 'Run simple test'
+        run: |
+             source $IPDK_RECIPE/scripts/dpdk/setup_env.sh $IPDK_RECIPE $SDE_INSTALL $DEPEND_INSTALL
+             sudo $IPDK_RECIPE/scripts/dpdk/copy_config_files.sh $IPDK_RECIPE $SDE_INSTALL
+             sudo $IPDK_RECIPE/scripts/dpdk/set_hugepages.sh
+             sudo env PATH="$PATH" HOME="$HOME" LD_LIBRARY_PATH="$LD_LIBRARY_PATH" SDE_INSTALL="$SDE_INSTALL" IPDK_RECIPE="$IPDK_RECIPE" DEPEND_INSTALL="$DEPEND_INSTALL" python3 $P4C_DIR/backends/dpdk/run-dpdk-ptf-test.py $P4C_DIR/testdata/p4_16_samples/add_on_miss0.p4 -p4c $P4C_DIR --ipdk-recipe $IPDK_RECIPE --testfile $P4C_DIR/testdata/p4_16_samples/add_on_miss0.py -ll DEBUG
```

Comment:
These env variables should be parameters to the `run-dpdk-ptf-test.py` script.

Reply:
Also going forward, we will encode these tests as part of CMake. This explicit test should not be part of the CI setup.